source/sass/31-molecules/map/_leaflet.scss

```
@@ -65,7 +65,7 @@
 
   .leaflet-control-zoom {
     a {
-      box-shadow: 0 8px 12px 0 rgba(0, 125, 179, .06);
+      box-shadow: 0 3px 14px rgba(0, 0, 0, 0.4);
```

Remove leading zeros.
```suggestion
      box-shadow: 0 3px 14px rgba(0, 0, 0, .4);
```